src/component/toolbox/ToolboxView.js

```diff
@@ -147,8 +149,36 @@ export default echarts.extendComponentView({
                 icons[featureName] = icon;
                 titles[featureName] = title;
             }
+
             var iconPaths = featureModel.iconPaths = {};
             zrUtil.each(icons, function (iconStr, iconName) {
+                // `tiled` is just for changing stack toggle icon, no need to add a path.
+                if (isMagicType && iconName === 'tiled') {
+                    return;
```

Comment:
It's better to put this logic in the `render` and `updateView` method of `MagicType.js`. ToolboxView should be not aware of what features it may have.

brush feature is a typical example: https://github.com/apache/incubator-echarts/blob/next/src/component/toolbox/feature/Brush.ts#L47

You may refactor it in the next branch if you wish